src/lib/import_export/csv/csv_parser.cpp

```diff
@@ -71,32 +72,29 @@ std::shared_ptr<Table> CsvParser::parse(const std::string& filename, const std::
     content_view = content_view.substr(field_ends.back() + 1);
 
     // create and start parsing task to fill chunk
-    tasks.emplace_back(std::make_shared<JobTask>([this, relevant_content, field_ends, &table, &segments]() {
-      _parse_into_chunk(relevant_content, field_ends, *table, segments);
-    }));
-    tasks.back()->schedule();
-  }
 
-  Hyrise::get().scheduler()->wait_for_tasks(tasks);
```

Comment:
I rather comment here so that the review is self-contained:

Could you clarify how this made testing difficult? The way I see it, the tasks do not leave the method, so how does multithreading have an externally visible effect?